Library/Homebrew/completions/fish.erb

```diff
@@ -153,7 +145,9 @@ function __fish_brew_suggest_casks_outdated -d "Lists outdated casks with the in
 end
 
 function __fish_brew_suggest_taps_installed -d "List all available taps"
-    brew tap
+    command find (brew --prefix)/Library/Taps -mindepth 2 -maxdepth 2 -type d \
+    | string replace homebrew- "" \
+    | string replace (brew --prefix)/Library/Taps/ ""
```

Comment:
```fish
❯ hyperfine 'brew tap' 'command find (brew --prefix)/Library/Taps -mindepth 2 -maxdepth 2 -type d    | string replace homebrew- "" | string replace (brew --prefix)/Library/Taps/ ""' --shell fish --warmup 1
Benchmark 1: brew tap
  Time (mean ± σ):     455.4 ms ±  11.8 ms    [User: 240.9 ms, System: 102.5 ms]
  Range (min … max):   441.5 ms … 474.4 ms    10 runs

Benchmark 2: command find (brew --prefix)/Library/Taps -mindepth 2 -maxdepth 2 -type d    | string replace homebrew- "" | string replace (brew --prefix)/Library/Taps/ ""
  Time (mean ± σ):      18.7 ms ±   1.2 ms    [User: 6.1 ms, System: 8.3 ms]
  Range (min … max):    16.7 ms …  23.0 ms    94 runs

Summary
  'command find (brew --prefix)/Library/Taps -mindepth 2 -maxdepth 2 -type d    | string replace homebrew- "" | string replace (brew --prefix)/Library/Taps/ ""' ran
   24.40 ± 1.72 times faster than 'brew tap'
```

Reply:
@razvanazamfirei Is this a strategy that could be used in bash/zsh completions too? May want to split it out into a command like `brew formulae` if so.

Reply:
@apainintheneck, I made the change. Let me know if this fixes it and if you think I should add `2>/dev/null` to the find command.

Reply:
Yep, this fixed it. Thanks. I'm not sure if we need to redirect stderr to /dev/null right now. It shouldn't print to stderr at all right?